Test FSIapp is failing

~~~
======================================================================
FAIL: test_mvqn_recursive_accelerator_variable_stiffness (convergence_accelerator_spring_test.ConvergenceAcceleratorSpringTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philipp/software/Kratos_prod/applications/FSIapplication/tests/convergence_accelerator_spring_test.py", line 245, in test_mvqn_recursive_accelerator_variable_stiffness
    self.test_mvqn_recursive_accelerator(forceA,forceB,solution)
  File "/home/philipp/software/Kratos_prod/applications/FSIapplication/tests/convergence_accelerator_spring_test.py", line 184, in test_mvqn_recursive_accelerator
    self.test_accelerator(force1,force2,solution,mvqn_recursive_settings)
  File "/home/philipp/software/Kratos_prod/applications/FSIapplication/tests/convergence_accelerator_spring_test.py", line 173, in test_accelerator
    self.assertAlmostEqual(expected_x[i],x_guess[i],delta=self.assert_delta)
AssertionError: 0.3333333333333335 != -2.0732226561216084e+88 within 1e-07 delta

Stdout:

Testing accelerator:  MVQN_recursive
~~~

it only fails sometimes, guess it has to do with initialization

--- a/applications/FSIapplication/tests/fsi_coupling_interface_test.py
+++ b/applications/FSIapplication/tests/fsi_coupling_interface_test.py
@@ -35,8 +35,10 @@ class FSICouplingInterfaceTest(UnitTest.TestCase):
         model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISPLACEMENT)
 
         # Import model part from mdpa file.
-        input_filename = 'FSIProblemEmulatorTest/test_FSI_emulator_Structural'
-        KratosMultiphysics.ModelPartIO(input_filename).ReadModelPart(model_part)
+        work_folder = "FSIProblemEmulatorTest"
+        input_filename = 'test_FSI_emulator_Structural'
+        with UnitTest.WorkFolderScope(work_folder,__file__):
+            KratosMultiphysics.ModelPartIO(input_filename).ReadModelPart(model_part)
 
         # Create the convergence accelerator
         conv_acc_factory = KratosMultiphysics.Parameters("""
@@ -69,7 +71,7 @@ class FSICouplingInterfaceTest(UnitTest.TestCase):
 
     def tearDown(self):
         with UnitTest.WorkFolderScope(self.work_folder, __file__):
-            KratosUtilities.DeleteFileIfExisting('FSIProblemEmulatorTest/test_FSI_emulator_Structural.time')
+            KratosUtilities.DeleteFileIfExisting('test_FSI_emulator_Structural.time')
 
     def checkResults(self):
         # Check the previously performed update

